### DISCUSSION TITLE
Introduction: Boaz Feldman

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#168] [Introductions]
Hey @theo-cox, @rufuspollock @Liyubov
Delighted to meet you all here - thanks you kindly Rufus for the invitation!

A few short words of introduction, I love joy, depth, multi-level thinking and the embodied experience of fractal perceptions and behaviours. I work as a clinical psychologist and contemplative, and slowly broadening my engagement towards social and political activism. I ordained as a Buddhist monk in my early 20's, and since have kept up a committed silent practice daily and annually in retreats. I've now built an NGO in Switzerland with a little about 25 collaborators, offering capacity building trainings in clinical trauma healing and community empowerment. I've worked for most major humanitarian organizations in conflict-affected regions and emergency relief.

Here's 

- my personal website: http://boazfeldman.com/
- my organizational website: https://neurosystemics.org/

I'm interested in furthering a systemitization of my method for resiliency and empowerment for social and political change in an upcoming PhD/PxD, and supporting the uprising and networking of communities for bottom-up political movement, heart-centered civil society movements and other such initiatives.

Looking forward to delving into the topic of community here with you all and getting to know you!

May it be for the benefit of all beings.

**Comment by rufuspollock** [comment]
@BoazBF very much welcome 😄

**Reply by theo-cox** [reply]
Great to be introduced Boaz! Have heard lots of great things so excited to connect further

**Comment by BoazBF** [comment]
Thx friends, see you soon - off to silence for now until Feb 2023
